Makefile

```diff
@@ -253,9 +265,11 @@ rm -rf $$TMP_DIR ;\
 endef
 
 .PHONY: bundle
+bundle: export RELATED_IMAGE_LIMITADOR := $(RELATED_IMAGE_LIMITADOR)
```

Comment:
I do not think it is necessary to export `RELATED_IMAGE_LIMITADOR` as env var. It is enough to have it as Makefile var. For `yq` the value will be an env var, but called `V`